pallets/parachain-staking/src/lib.rs

```diff
@@ -1982,18 +1984,19 @@ pub mod pallet {
 			candidate: &T::AccountId,
 			more: BalanceOf<T>,
 		) -> Result<bool, sp_runtime::DispatchError> {
-			ensure!(
-				!Self::delegation_request_revoke_exists(candidate, delegator),
-				Error::<T>::PendingDelegationRevoke
-			);
-			let mut state = <DelegatorState<T>>::get(delegator).ok_or(Error::<T>::DelegatorDNE)?;
-			state.increase_delegation::<T>(candidate.clone(), more)
-		}
+			let in_top = Self::delegation_bond_more_without_event(
+				delegator.clone(),
+				candidate.clone(),
+				more.clone(),
+			)?;
+			Pallet::<T>::deposit_event(Event::DelegationIncreased {
+				delegator: delegator.clone(),
+				candidate: candidate.clone(),
+				amount: more,
+				in_top,
+			});
 
-		fn is_delegation_exist(delegator: &T::AccountId, candidate: &T::AccountId) -> bool {
-			<DelegatorState<T>>::get(delegator)
-				.map(|state| state.get_bond_amount(candidate).is_some())
-				.unwrap_or(false)
+			Ok(in_top)
```

Comment:
What is this function used for?

Its reference was at line 1342 but deleted.

Reply:
The `is_delegation_exist` function is no longer necessary. 

Previously, it was used in OAK-blockchain to check for the existence of delegation before calling `delegator_bond_more`, in order to prevent an `InsufficientBalance error` from being thrown prematurely.